app/segments/(team)/chooseTeam.js

```diff
@@ -64,7 +92,7 @@ function ChooseTeam() {
         }}
         refreshControl={<RefreshInvalidate invalidateKeys={invalidateKeys} />}
```

Comment:
Don't know if it was PR but when I refresh this screen after requesting to join the team, there is an animation that takes me to the same page. Might be worth looking into more.

Reply:
I think this is in IWB-4 and I've just pushed a fix. Can you check it out again?